packages/runner/package.json

```diff
@@ -36,6 +37,8 @@
     "prop-types": "15.7.2",
     "react": "16.8.6",
     "react-dom": "16.8.6",
+    "react-popper": "2.2.5",
+    "react-shadow-dom-retarget-events": "1.0.11",
     "rimraf": "3.0.2",
     "webpack": "^4.44.2",
```

Comment:
Can you update the runner's README as well?

It's doing a few more things than it currently lists with this merge:
- Contains `dom.js`, which uses proprietary webpack loaders and cannot easily be imported with Vite (dev server in `@packages/app`). Once `dom.js` is free of webpack-specific loader code, we should move it to `@packages/app`.
- Contains UI code for Cypress Studio, which was marked as experimental in Cypress 9.x and won't be part of Cypress 10.x initially. It will return at a later date. Until then, the code will be here. It's not currently used in the app.
- contains Legacy Cypress styles, most of these can likely be removed.

out of date:
- Some existing tests in `cypress/e2e` should be migrated to `@packages/app/cypress/e2e/runner`.

Reply:
Done. I changed the `UI code for Cypress Studio` with `Cypress Studio Recorder code`. Because I removed `react` UI code in this PR and `Studio Recorder` is currently injected into `app`.